CONTRIBUTING.md

```diff
@@ -3,12 +3,20 @@ Contributing to Coq LSP
 
 Thank you very much for willing to contribute to coq-lsp!
 
-`coq-lsp` is in alpha stage, the first step for contributing is to get
-in touch with the devs at Zulip.
+`coq-lsp` has two components:
+
+- a `coq-lsp` server written in OCaml, which is a very standard OCaml
+  dune project.
+
+- a `coq-lsp` VS Code extension written in TypeScript, and setup as a
+  standard npm package using `esbuild` as the bundler. The extension
+  has a main component and some webviews components written using
+  React.
```

Comment:
```suggestion
- A VS Code extension written in TypeScript.
```

Reply:
We don't need all this technical detail in the beginning.

Reply:
Ok, I've done like that; there is still some duplication between CONTRIBUTING and README, I'm unsure where things should go.